### DISCUSSION TITLE
Is it possible build googletest without Visual Studio?

### DISCUSSION THREAD

**Discussion by StanleyWu8787** [google/googletest#4597] [Community Help] [answered]
I follow https://github.com/google/googletest/blob/main/googletest/README.md

There is a part about windows:
```
If you use Windows and have Visual Studio installed, a gtest.sln file and several .vcproj files will be created. You can then build them using Visual Studio.
```

My question is can we build googletest without Visual Studio?
There is no any information about this question.

And there is no `Makefile` after `cmake ..`

Thanks for your help!

**Comment by Shriharsh-Deshmukh** [comment, marked as answer]
Yes, you can build GoogleTest without Visual Studio by using CMake with a different generator. For example, on Windows, you can use MinGW or another compatible toolchain.

1. **Generate Makefiles:**
   ```bash
   cmake -G "MinGW Makefiles" ..
   ```

2. **Build Using MinGW:**
   ```bash
   mingw32-make
   ```

This process avoids using Visual Studio and relies on MinGW to build GoogleTest.

**Reply by StanleyWu8787** [reply]
Thanks for answer.
